dist/index.html

```diff
@@ -1,37 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Lora&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
-    />
-    <link rel="stylesheet" href="/src/style.css">
-
-    <title>Movies Database</title>
-  <script defer src="main.js"></script></head>
-  <body>
-    <header id="navigation">
-      <a href="#" class="logo">LOGO</a>
-      <a href="#" class="logo">Latest Movies/Shows</a>
-      <nav id="other-interfaces">
-        <a href="#" class="interface">Tv Shows</a>
-        <a href="#" class="interface">Movies</a>
-      </nav>
-    </header>
-    
- 
-    <footer id="footer" class="attribution">
-
-        <p>Created by <a href="https://github.com/bengigo">bengigo</a>,
-            <a href="https://github.com/29td">Nonhlanhla</a> & <a href="https://github.com/Therealadetunji">Michael</a>
-        </p>
-    </footer>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css2?family=Lora&display=swap" rel="stylesheet" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="/src/style.css">
+
+  <title>Movies Database</title>
+<script defer src="main.js"></script></head>
+
+<body>
+  <header id="navigation">
+    <a href="#" class="logo">LOGO</a>
+    <a href="#" class="logo">Latest Movies/Shows</a>
+    <nav id="other-interfaces">
+      <a href="#" class="interface">Tv Shows</a>
+      <a href="#" class="interface">Movies</a>
+    </nav>
+  </header>
+
+  <section id="list">
+    <!-- template for dynamic display -->
+    <!-- <div class="show">
+      <img class="poster" src="">
+      <h2 class="name">What a Show!</h2>
+      <p class="genre">thriller</p>
+      <button class="btn-comments">comments</button>
+      <button class="btn-reservations">reservations</button>
+    </div> -->
```

Comment:
### Suggestions 
Your index.html file would look more professional if you removed the commented part

_Cheers Happy coding!_

Reply:
Thank you. I removed the unnecessary comments 👍